apps/frontend/hub/src/lib/views/catalog/__tests__/category_load.test.ts

```
@@ -247,18 +247,18 @@ describe('category load — C-396 AC-2 (static index, no Postgres)', () => {
-    const data = await load({
+    const data = (await load({
       params: { category: 'lpc' },
       setHeaders,
       depends,
-    } as never);
+    } as never)) as Awaited<ReturnType<typeof load>>;
 
     expect(data).toBeDefined();
-    expect(data.category).toBe('lpc');
-    expect(data.entries).toHaveLength(2);
+    expect(data!.category).toBe('lpc');
```

Biome (2.5.7)

[error] Forbidden non-null assertion.

(lint/style/noNonNullAssertion)